src/components/PostDetails.tsx

```diff
@@ -98,21 +62,21 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
       </div>
 
       <div className="block">
-        {!loaded && <Loader />}
+        {!loading && <Loader />}
```

Comment:
The condition for displaying the `Loader` seems inverted. It should be shown when `loading` is true, not false. Consider changing the condition to `{loading && <Loader />}`.